src/PhpEnvironment/Response.php

```diff
@@ -29,6 +30,11 @@ class Response extends HttpResponse
      */
     protected $contentSent = false;
 
+    /**
+     * @var callable
```

Comment:
```suggestion
     * @var null|callable
```

Because it's not set by default.

Reply:
implemented.